Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -0,0 +1,1150 @@
+
+        if not alerts_output:
+            demisto.debug('API returned no alerts from search api')
+            return next_run, incidents
+
+        alerts = alerts_output['Alert']
+
+        for alert in alerts:
+            guid = alert['ID']
+            alert_time = alert['Time']
+            incident = {
+                'name': f'Varonis alert {guid}',
+                'occurred': f'{alert_time}Z',
+                'rawJSON': json.dumps(alert),
+                'type': 'Varonis DSP Incident',
+                'severity': convert_to_demisto_severity(alert['Severity']),
+            }
+
+            incidents.append(incident)
+
+        if len(alert_guids) < max_results:
+            return next_run, incidents
+
+    return next_run, incidents
+
+
+def varonis_get_alerts_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get alerts from Varonis DA
+
+    :type client: ``Client``
+    :param client: Http client
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['threat_model_name']`` List of requested threat models to retrieve
+        ``args['page']`` Page number (default 1)
+        ``args['max_results']`` The max number of alerts to retrieve (up to 50)
+        ``args['Start_time']`` Start time of the range of alerts
+        ``args['End_time']`` End time of the range of alerts
+        ``args['Alert_Status']`` List of required alerts status
+
+    :return:
+        A ``CommandResults`` object
+
+    :rtype: ``CommandResults``
+    """
+    threat_model_names = args.get('threat_model_name', None)
+    max_results = args.get('max_results', None)
+    start_time = args.get('Start_time', None)
+    end_time = args.get('End_time', None)
+    alert_statuses = args.get('Alert_Status', None)
+    page = args.get('page', '1')
+
+    threat_model_names = try_convert(threat_model_names, lambda x: argToList(x))
+    max_results = try_convert(
+        max_results,
+        lambda x: int(x),
+        ValueError(f'max_results should be integer, but it is {max_results}.')
+    )
+    start_time = try_convert(
+        start_time,
+        lambda x: datetime.fromisoformat(x),
+        ValueError(f'start_time should be in iso format, but it is {start_time}.')
+    )
+    end_time = try_convert(
+        end_time,
+        lambda x: datetime.fromisoformat(x),
+        ValueError(f'end_time should be in iso format, but it is {start_time}.')
+    )
+
+    alert_statuses = try_convert(alert_statuses, lambda x: argToList(x))
+    page = try_convert(
+        page,
+        lambda x: int(x),
+        ValueError(f'page should be integer, but it is {page}.')
+    )
+
+    result = varonis_get_alerts(client, alert_statuses, threat_model_names, start_time, end_time, max_results, page, None)
+    outputs = create_output(ALERT_OUTPUT, result['rows'])
+    page_size = result['rowsCount']
+    outputs = enrich_with_pagination(outputs, page, page_size)
```

Comment:
https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown

Reply:
@kgal-pan added readable output to result